packages/image_picker/pubspec.yaml

```
@@ -0,0 +1,29 @@
+name: image_picker_tizen
+description: Flutter plugin for selecting images from the Tizen image
+  library, and taking new pictures with the camera.
+version: 0.0.1
```

```suggestion
version: 1.0.0
```
For consistency with CHANGELOG.md.

---

done.